Marlin/language_nl.h

```diff
@@ -35,33 +35,39 @@
 #define WELCOME_MSG                         MACHINE_NAME " gereed."
 #define MSG_SD_INSERTED                     "Kaart ingestoken"
 #define MSG_SD_REMOVED                      "Kaart verwijderd"
+#define MSG_LCD_ENDSTOPS                    "Endstops" // Max length 8 characters
```

Comment:
Since I have max 8 chars I rather leave it as Endstops. Eindstops is the Dutch variant so almost the same!